src/bp/ui-admin/src/Pages/Workspace/Bots/CreateBotModal.tsx

```diff
@@ -73,6 +75,7 @@ class CreateBotModal extends Component<Props, State> {
   componentDidMount() {
     this.loadCategories()
     this.loadTemplates()
+    this.props.fetchBots()
```

Comment:
I'm not sure fetchBots is necessary here, since the list of bots is already refreshed by its parent